scroll.js

```diff
@@ -0,0 +1,23 @@
+window.onscroll = function() {scrollFunction()};
+
+function scrollFunction() {
+
+    let logoContainer = document.querySelector(".logo")
```

Comment:
logoContainer is not used anywhere, so you should remove it.

Also in this case is better to use const instead of let since values are not going to change.